website/src/_posts/generics-by-examples.md

```diff
@@ -169,6 +169,29 @@ This will get you rid of the following error:
 
 > Class Bar extends generic class Collection but does not specify its types: T
 
+Specify template type variable of a generic trait when using it
+------------------------
+
+The `@use` tag needs to be used instead of `@implements` or `@extends`:
+
+```php
+/** @use Part<Foo> */
+class Bar
+{
+    use Part;
+}
+```
+
+The `@use` tag can be defined on the PHPDoc of the class or the use clause. Both work the same way:
+
+```php
+class Bar
+{
+    /** @use Part<Foo> */
```

Comment:
1) Please put it as a last example
2) Please show only `@use` above `use` so I can merge it now.
3) After your PR with `@use` above class is merged, send another PR to update the documentation.

Thank you!

Reply:
Hi there!

1. Put it as last example: done, with a little more complete example.
2. Show only `@use` above `use`: done.
3. `@use` above class : commit ready into another branch, I will PR once https://github.com/phpstan/phpstan-src/pull/2638 is merged.

Thank you!